src/docs/asciidoc/advancedFeatures.adoc

```diff
@@ -716,9 +734,112 @@ as the library is usually upstream of the client code.
 The tool for reading data from the context is the static `Mono.subscriberContext()`
 method.
 
-Consider an example of a library reading information from the `Context`: A reactive
-HTTP client that takes a `Mono<String>` as the source of data for a `PUT` but also looks
-for a particular Context key to add a correlation ID to the request's headers.
+=== Simple Examples
+The examples in this section are meant as ways to better understand some of the caveats of
+using a `Context`.
+
+Let's first look back at our simple example from the introduction in a bit more details:
+
+[source,java]
+----
+String key = "key";
+Mono<String> r = Mono.just("Hello")
+                .flatMap( s -> Mono.subscriberContext() //<2>
+                                   .map( ctx -> s + " " + ctx.get(key))) //<3>
+                .subscriberContext(ctx -> ctx.put(key, "World")); //<1>
+
+StepVerifier.create(r)
+            .expectNext("Hello World") //<4>
+            .verifyComplete();
+----
+<1> The chain of operators ends with a call to `subscriberContext(Function)` that puts
```

Comment:
The `<1>` implies this, but it might be valuable to make it more explicit that even though `subscriberContext` is the last piece of code, it is the first code that is executed. It might be valuable to do the same with the the other steps.